docker-compose.yml

```diff
@@ -11,11 +11,50 @@ services:
     ports:
       - "8080:8080"
     networks:
-      - pagopa-ecommerce-net
+      - pagopa-ecommerce-helpdesk-net
+  oracle:
+    container_name: pagopa-ecommerce-helpdesk-oracle
+    build: ./docker/oracleDB/liquibase
+    profiles:
+      - arm64
+    image: pagopadcommonacr.azurecr.io/dboracle-ee-12.2.0.1-slim
+    ports:
+      - 1521:1521
+      - 5500:5500
+    expose:
+      - '1521'
+      - '5500'
+    extra_hosts:
+      - "spid-testenv2:127.0.0.1"
+      - "spid-testenv2-2:127.0.0.1"
+      - "pagopa-dev:127.0.0.1"
+      - "spid-validator:127.0.0.1"
+  mongo:
+    container_name: pagopa-ecommerce-helpdesk-mongo
+    image: mongo
+    env_file: docker/mongoDB/mongodb.env
+    ports:
+      - "27017:27017"
+    networks:
+      - pagopa-ecommerce-helpdesk-net
+    healthcheck:
+      test: echo "db.stats().ok" | mongo --quiet
+      interval: 10s
+
+  mongo-express:
+    container_name: pagopa-ecommerce-helpdesk-express
+    image: mongo-express
+    env_file: docker/mongoDB/mongo-express.env
+    ports:
+      - "8081:8081"
+    networks:
+      - pagopa-ecommerce-helpdesk-net
+    depends_on:
+      - mongo
 
 networks:
-  pagopa-ecommerce-net:
-    name: pagopa-ecommerce-net
+  pagopa-ecommerce-helpdesk-net:
+    name: pagopa-ecommerce-helpdesk-net
```

Comment:
Why put it in a different network?

Reply:
Ops, I modified it in one of the many local rest to make it working with M1 arch. Restored with https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/pull/7/commits/ce7f733510bba02bffbf3d7da28f7a69d0a35978